scout/adapter/mongo/variant_loader.py

```diff
@@ -543,6 +548,44 @@ def _load_variants(
 
         return nr_inserted
 
+    def _is_causative_other(
+        self,
+        variant: cyvcf2.Variant,
+        category: str = "snv",
+        build: str = "37",
+    ) -> bool:
+        """Check if variant is on the list of causatives from other cases.
```

Comment:
Also from other customers right?

Reply:
Indeed. Potentially really important variant if it is marked causative by other institutes. Also it will not be flagged on other institutes cases until they start collaborating, but that it may feel unexpected with the variant missing if they do.

Reply:
Agreed